inspect

```diff
@@ -0,0 +1,127 @@
+import os
+import json
+from os import listdir
+import os
+
+def inspectSummary(tempdir, newName):
+    os.chdir(os.path.join(tempdir + '/' + newName))
+
+    openRead = open('SUMMARY', 'r')
+    wantedValues = ['BOARD', 'FIRMUX', 'BUILD', 'LOAD', 'RAM', 'print', 'UPTIME']
+    summaryFileContent = []
+
+    for line in openRead:
+        for value in wantedValues:
+
+            if value in line:
+                summaryFileContent.append(line)   
+
+    dict = {}
```

Comment:
`dict` kintamasis nieko nepasako apie jį patį. Skaitant kodą jo neparašiusiam neaišku ka šis kintamasis daro. Reikėtų jį pervadinti pagal paskirti pvz.: `allSysInfo` arba `systemStats`.